### DISCUSSION TITLE
scrolling columns when no results

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#46] [Shipped - Live on production]
I noticed some behaviour that could be improved. When having a list view on screen (with more columns than fitting on the screen) and searching some device in the search window it happens that you get zero results because one of the columns has a filter enabled. And that filter might be on a column that is outside the current view so requiring scrolling the view to the right column. However, when there are zero results it is not possible to scroll. So you have to remove the search words, then find the column that has a filter, and then search again. Or reset the filter settings. 

It would be nicer if scrolling was also allowed/possible when having zero search results in the table.

**Comment by YvesR** [comment]
This is a glitch of the front end component. In this case we might need moveTo 0,0 of the view point. Need to discuss with dev team.

**Reply by YvesR** [reply]
We fixed it. The header is now scrollable in such a situation so you can scroll to adjust.
